app/views/components/gds/gds_main_template.scala.html

```diff
@@ -89,6 +95,12 @@
         )))
     </div>
     @contentBlock
+    @hmrcReportTechnicalIssue(
+        ReportTechnicalIssue(
+            serviceCode = appConfig.keyStoreSource,
```

Comment:
Nit: would be nicer if 'keyStoreSource' was renamed to 'appName' like in internal-frontend. Seems odd having a persistence var here (i know values are the same)

Reply:
@glidester your wish has come true.

Reply:
Resolved